install.sh

```diff
@@ -1,184 +1,133 @@
 #!/usr/bin/env bash
-set -euo pipefail
 
-echo "Let's get you set up with Rustlings!"
-
-echo "Checking requirements..."
-if [ -x "$(command -v git)" ]
-then
-    echo "SUCCESS: Git is installed"
-else
-    echo "ERROR: Git does not seem to be installed."
-    echo "Please download Git using your package manager or over https://git-scm.com/!"
-    exit 1
-fi
-
-if [ -x "$(command -v cc)" ]
-then
-    echo "SUCCESS: cc is installed"
-else
-    echo "ERROR: cc does not seem to be installed."
-    echo "Please download (g)cc using your package manager."
-    echo "OSX: xcode-select --install"
-    echo "Deb: sudo apt install gcc"
-    echo "Yum: sudo yum -y install gcc"
-    exit 1
-fi
+# Compare two version strings, v1 and v2, passed as arguments (e.g 1.31 and
+# 1.33.0). Returns 1 if v1 > v2, 0 if v1 == v2, 2 if v1 < v2.
+compare_versions() {
+    [ "$1" == "$2" ] && return 0
+    declare -a v1=() v2=()
+    mapfile -d . -t v1 < <(echo "$1")
```

Comment:
Hm, this fails on my M1 Pro Mac on the default Apple bash shell because it seems `mapfile` is not provided.

Reply:
Ahh, sorry, I was tidying up while preparing to install a new distro and forgot that you would get a notification when I pushed to the branch. I didn't verify that the script still worked; I just pushed what I had. I wouldn't have caught your particular issue, though, so thanks for bringing it up!

If you like, I can revise this properly once more so that we can (hopefully) get it merged!